Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -532,8 +532,10 @@ def audit_conflicts
       end
     end
 
+    versioned_conflicts_whitelist = %w[node@ bash-completion@].freeze
```

Comment:
I think it would make sense to move all of these whitelists outside, since they really are supposed to be constants, and it makes the methods easier to read if there aren't a billion lines of constants in-between “actual“ code.